docker-compose.yml

```diff
@@ -36,11 +37,12 @@ services:
             - BEHAVIOR_TCHCLIENT=client,server,encoding
             - BEHAVIOR_TCHSERVER=client,server,encoding
             # BEHAVIOR_HTTPCLIENT TODO
-            # BEHAVIOR_HTTPSERVER TODO
+            - BEHAVIOR_HTTPSERVER=client,httpserver
```

Comment:
I don't understand the name of this behavior? What are we testing here exactly...?